en/docs/security-guidelines/secure-engineering-guidelines/owasp-csrf-guard.md

```diff
@@ -263,7 +263,7 @@ Following property can be enabled to invalidate the user session if an CSRF atta
 ## WSO2 Product Integration Checklist
 
```

Comment:
let's not have consecutive headings without text in between

Reply:
Fixed in daaff6e35f88c078e7d1e855078a5777cee5ee83